internal/handlers/telegram/handler.go

```diff
@@ -126,7 +146,12 @@ documentHandler receives a document object from Telegram, and sends
 a notification to IRC.
 */
 func documentHandler(tg *Client, u *tgbotapi.Message) {
-	username := GetUsername(u.From)
+	username := ""
+	if tg.IRCSettings.ShowZWSP {
+		username = ZwspUsername(u.From)
+	} else {
+		username = GetUsername(u.From)
+	}
```

Comment:
This code feels repetitive since it is repeated many times over in the other handlers. This adds additional testing overhead from my point-of-view. Is it possible to add a helper function somewhere else that will return the correct username based on whether the Zwsp setting is enabled?

Reply:
I actually thought for a while about just moving this into a separate helper function in the `helpers.go` file. My _only_ reasoning for doing it this way was it requires adding several more layers of complexity into the helpers file, but it definitely makes the `handler.go` file much cleaner, so I will look into splitting that out more.

Reply:
@Tjzabel Got it, makes sense. I would be OK addressing this in a future PR if it is complex to do right now. I would rather get this merged and deployed, and take time for polish and small improvements once we hit a regular cadence of meetings again.